src/Diagnostics.Logger/ETWProvider/DiagnosticsETWProvider.cs

```diff
@@ -140,6 +140,16 @@ public void LogRuntimeHostDetectorAscInsight(string RequestId, string Source, st
                 Details);
         }
```

Comment:
Do you need this anymore since your new event is essentially a superset of this event?

Reply:
Don't really need it, it was not working, so I figured easiest thing to do was just to create a new event